packages/e2e/.detoxrc.json

```diff
@@ -0,0 +1,57 @@
+{
+  "skipLegacyWorkersInjection": true,
+  "apps": {
+    "ios.debug": {
+      "type": "ios.app",
+      "binaryPath": "../../examples/react-native/ios/build/Build/Products/Debug-iphonesimulator/ReactNative.app",
+      "build": "xcodebuild -workspace ../../examples/react-native/ios/ReactNative.xcworkspace -scheme ReactNative -configuration Debug -sdk iphonesimulator -arch x86_64 -derivedDataPath ../../examples/react-native/ios/build"
+    },
+    "ios.release": {
+      "type": "ios.app",
+      "binaryPath": "../../examples/react-native/ios/build/Build/Products/Release-iphonesimulator/ReactNative.app",
+      "build": "xcodebuild -workspace ../../examples/react-native/ios/ReactNative.xcworkspace -scheme ReactNative -configuration Release -sdk iphonesimulator -arch x86_64 -derivedDataPath ../../examples/react-native/ios/build"
+    },
+    "android.debug": {
+      "type": "android.apk",
+      "binaryPath": "../../examples/react-native/android/app/build/outputs/apk/debug/app-debug.apk",
+      "build": "cd ../../examples/react-native/android && ./gradlew assembleDebug assembleAndroidTest -DtestBuildType=debug && cd ../../.."
+    },
+    "android.release": {
+      "type": "android.apk",
+      "binaryPath": "../../examples/react-native/android/app/build/outputs/apk/release/app-release.apk",
+      "build": "cd ../../examples/react-native/android && ./gradlew assembleRelease assembleAndroidTest -DtestBuildType=release && cd ../../.."
+    }
+  },
+  "devices": {
+    "simulator": {
+      "type": "ios.simulator",
+      "device": {
+        "type": "iPhone 13"
```

Comment:
Most popular according to [Mixpanel](https://mixpanel.com/trends/#report/iphone_models/from_date:-932,report_unit:month,to_date:0)